packaging/debian/control

```diff
@@ -0,0 +1,19 @@
+Source: libsxg
+Priority: optional
+Maintainer: Hiroki Kumazaki <kumagi@google.com>
+Build-Depends: debhelper (>= 11)
+Standards-Version: 4.1.3
+Section: libs
+Homepage: https://github.com/google/libsxg
+
+Package: libsxg-dev
+Section: libdevel
+Architecture: any
+Multi-Arch: same
+Depends: libssl-dev (>=1.1.1c), ${misc:Depends}
+Description: Signed HTTP Exchange toolkit library.
+ Signed HTTP Exchange(SXG) is file format which contains an HTTP exchange
```

Comment:
Put a space before the left parentheses: `Signed HTTP Exchange (SXG)`.

Reply:
Done.